.yamato/training-int-tests.yml

```diff
@@ -14,8 +14,6 @@ test_linux_training_int_{{ editor.version }}:
       sudo apt-get update && sudo apt-get install -y python3-venv
       python3 -m venv venv && source venv/bin/activate
       python -m pip install pyyaml --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple
-      python -m pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
```

Comment:
This was needed before (when we did the backwards compatibility in the same test) but not anymore. Shaves 1-2 minutes off the test time.